cmd/cstor-pool-mgmt/crdops/controller-pool.go

```diff
@@ -0,0 +1,357 @@
+/*
```

Comment:
Have you run against any tool to validate the folder & file naming conventions

Reply:
Can you please suggest one for file-folder name validation?